crates/biome_dependency_graph/src/dependency_graph.rs

```diff
@@ -121,8 +121,19 @@ impl DependencyGraph {
             added_or_updated_paths,
             removed_paths,
         ));
-        let resolver =
-            ResolverGeneric::new_with_cache(resolver_cache.clone(), ResolveOptions::default());
+        let resolve_options = ResolveOptions {
+            enforce_extension: EnforceExtension::Disabled,
+            extensions: vec![
+                ".js".into(),
+                ".jsx".into(),
+                ".ts".into(),
+                ".tsx".into(),
```

Comment:
Should we also add `cjs` and `mjs`? (And TypeScript counter parts)

Reply:
Yeah, good call